web/src/components/Plugin/locales/en-US.ts

```diff
@@ -41,9 +41,9 @@ export default {
   'component.pluginForm.cors.allow_headers.tooltip':
     'Which headers are allowed to set in request when access cross-origin resource. Multiple value use , to split. When allow_credential is false, you can use * to indicate allow all request headers. You also can allow any header forcefully using ** even already enable allow_credential, but it will bring some security risks.',
   'component.pluginForm.cors.expose_headers.tooltip':
-    'Which headers are allowed to set in response when access cross-origin resource. Multiple value use , to split.',
+    'Which headers are allowed to set in response when access cross-origin resource. Multiple value use , to split. When allow_credential is false, you can use * to indicate allow any header. You also can allow any header forcefully using ** even already enable allow_credential, but it will bring some security risks.',
```

Comment:
```
Which Origins is allowed to enable CORS, format as：scheme://host:port, for example: https://somehost.com:8081. Multiple origin use , to split. When allow_credential is false, you can use * to indicate allow any origin. you also can allow all any origins forcefully using ** even already enable allow_credential, but it will bring some security risks.
```
Please refer: https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/cors.md

Reply:
the docs updated~